mk/variables.mk

```diff
@@ -69,8 +79,27 @@ DOCKER_IMAGE_TAG ?= $(shell git rev-parse --short HEAD)
 # Compose the container image with all the above
 DOCKER_IMAGE ?= $(DOCKER_IMAGE_BASE):$(DOCKER_IMAGE_TAG)
 
-# KAFKA configurations
+#
+# Kafka configuration variables
+#
+
+# The directory where the kafka data will be stored
 KAFKA_DATA_DIR ?= $(PROJECT_DIR)/kafka/data
+
+# The directory where the kafka configuration will be
+# bound to the containers
 KAFKA_CONFIG_DIR ?= $(PROJECT_DIR)/kafka/config
-KAFKA_TOPICS ?= "repo-introspection"
 
+# The topics used by the repository
+# Updated to follow the pattern used at playbook-dispatcher
+KAFKA_TOPICS ?= platform.content-sources.introspect
+
+# The group id for the consumers; every consumer subscribed to
+# a topic with different group-id will receive a copy of the
+# message. In our scenario, any replica of the consumer want
+# only one message to be processed, so we only use unique
+# group id, so far
```

Comment:
```suggestion
# group ID.
```
so far ? "At the moment", "At present" , but not much point to say that if there is no info about when this will change.

Reply:
thanks mate! I have updated to `at the moment`.